anomaly_detector/anomaly_detector.py

```diff
@@ -187,8 +190,13 @@ def infer(self, false_positives=None):
                 s["predict_id"] = str(uuid.uuid4())
                 s["anomaly_score"] = dist[i]
                 # Record anomaly event in fact_store and
+                if false_positives is not None:
+                    if {"message": s["message"]} in false_positives:
+                        _LOGGER.info("False positive was found (score: %f): %s" % (dist[i], s["message"]))
+                        FALSE_POSITIVE_METRIC.labels(id=s["predict_id"], message=s["message"], score=dist[i]).inc()
```

Comment:
@zmhassan just remove the message and score label from the metric. Then it's just a counter of reported `FALSE_POSITIVES`

Reply:
Good point.

Reply:
Fixed it. Done !